vipps-login-api.md

```diff
@@ -185,6 +185,10 @@ Which of the flows to use is controlled with the initiation of the individual lo
 
 The flows are described below.
 
+See
```

Comment:
Not sure if I understood why you're referring to the landingpage (ecom) here?

Reply:
Maybe it was an error. I don't know why I put that there. I will remove it.